src/insexpand.c

```diff
@@ -7001,6 +7004,8 @@ setup_cpt_sources(void)
 	    idx++;
 	}
     }
+
+    vim_free(cpt);
```

Comment:
There are a few other returns above this one here where we shold also `vim_free(cpt);`.

Reply:
Thanks, fixed.